src/Features/Core/Portable/AddImport/SymbolReferenceFinder.cs

```diff
@@ -411,23 +424,13 @@ private async Task<ImmutableArray<SymbolReference>> GetReferencesForCollectionIn
             SearchScope searchScope, CancellationToken cancellationToken)
         {
             cancellationToken.ThrowIfCancellationRequested();
-            if (_owner.CanAddImportForMethod(_diagnosticId, _syntaxFacts, _node, out _) &&
+            if (_owner.CanAddImportForMember(_diagnosticId, _syntaxFacts, _node, out _) &&
                 !_syntaxFacts.IsSimpleName(_node) &&
-                _owner.IsAddMethodContext(_node, _semanticModel))
+                _owner.IsAddMethodContext(_node, _semanticModel, out var objectCreationExpression))
             {
-                var symbols = await searchScope.FindDeclarationsAsync(
```

Comment:
i realized the code for searching for extension .Add could use the existing GetReferencesForExtensionMethodAsync helper we already have (and use for looking up other well known named extnesions).